neptune/new/internal/init_impl.py

```diff
@@ -316,7 +316,8 @@ def init(project: Optional[str] = None,
         background_jobs.append(TracebackJob(traceback_path, fail_on_exception))
         background_jobs.append(PingBackgroundJob())
 
-    _run = Run(api_run.uuid, backend, operation_processor, BackgroundJobList(background_jobs))
+    _run = Run(api_run.uuid, backend, operation_processor, BackgroundJobList(background_jobs),
+               api_run.workspace, api_run.project_name, api_run.short_id)
```

Comment:
maybe you could just use `api_run, backend, operation_processor, BackgroundJobList(background_jobs)` args, since you need 4 out of 5 `ApiRun`'s attributes?

Reply:
Run is not connected to ApiRun in this way - It's easy to imagine that Run can exist without ApiRun. Also stuff like workspace and project_name are characteristics of Run, bu ApiRun is not a characteristic of Run